indexer/services/comlink/src/types.ts

```diff
@@ -57,6 +66,9 @@ export interface AddressResponse {
 
 export interface SubaccountResponseObject {
   address: string,
+  /**
+   * @isInt
+   */
```

Comment:
Wow this is so ugly. Is this just a workaround or do we need to do this for any integer in TS?

Reply:
yeah we need this for any integer in the documentation we're using